src/chef-mover/src/mover_server_admins_global_group_callback.erl

```diff
@@ -0,0 +1,210 @@
+         migration_action/2,
+         next_object/0,
+         error_halts_migration/0,
+         reconfigure_object/2,
+         needs_account_dets/0
+        ]).
+
+-include("mover.hrl").
+-include("mv_oc_chef_authz.hrl").
+
+-define(GLOBAL_PLACEHOLDER_ORG_ID, <<"00000000000000000000000000000000">>).
+
+-record(container, {authz_id}).
+-record(user, {authz_id}).
+-record(mover_chef_group, {
+          id,
+          org_id,	  
+          authz_id,
+          name,
+          last_updated_by,
+          created_at,
+          updated_at
+         }).
+
+
+migration_init() ->
+    mv_oc_chef_authz_http:create_pool(),
+    mover_transient_migration_queue:initialize_queue(?MODULE, [?GLOBAL_PLACEHOLDER_ORG_ID]).
+
+migration_action(GlobalOrgId, _AcctInfo) ->
+    BifrostSuperuserAuthzId = mv_oc_chef_authz:superuser_id(),
+    ErchefSuperuserAuthzId = get_erchef_superuser_authz_id(),
+    ServerAdminsAuthzId = case create_server_admins_authz_group(BifrostSuperuserAuthzId) of
+        AuthzId when is_binary(AuthzId) ->
+            AuthzId;
+        AuthzError ->
+            lager:error("Could not create new authz group for server-admins."),
+            throw(AuthzError)
+    end,
+    
+    case create_server_admins_global_group(ServerAdminsAuthzId, GlobalOrgId) of
+        {chef_sql, {GroupError, _}} ->
+	    lager:error("Could not create new erchef group for server-admins."),
+            throw(GroupError);
+	_ -> true
+    end,
+
+    %% Put pivotal in the global group
+    mv_oc_chef_authz:add_to_group(ServerAdminsAuthzId, actor, ErchefSuperuserAuthzId, BifrostSuperuserAuthzId),
+
+    %% Grant server-admins global group permissions on users container 
+    UserContainerAuthzId = get_user_container_authz_id(),
+    mv_oc_chef_authz:add_ace_for_entity(BifrostSuperuserAuthzId, group, ServerAdminsAuthzId, container, UserContainerAuthzId, read),
+    mv_oc_chef_authz:add_ace_for_entity(BifrostSuperuserAuthzId, group, ServerAdminsAuthzId, container, UserContainerAuthzId, create),
+    mv_oc_chef_authz:add_ace_for_entity(BifrostSuperuserAuthzId, group, ServerAdminsAuthzId, container, UserContainerAuthzId, update),
+    mv_oc_chef_authz:add_ace_for_entity(BifrostSuperuserAuthzId, group, ServerAdminsAuthzId, container, UserContainerAuthzId, delete),
+
+    %% Grant pivotal permissions on server-admins global group
+    mv_oc_chef_authz:add_ace_for_entity(BifrostSuperuserAuthzId, actor, ErchefSuperuserAuthzId, group, ServerAdminsAuthzId, create),
+    mv_oc_chef_authz:add_ace_for_entity(BifrostSuperuserAuthzId, actor, ErchefSuperuserAuthzId, group, ServerAdminsAuthzId, read),
+    mv_oc_chef_authz:add_ace_for_entity(BifrostSuperuserAuthzId, actor, ErchefSuperuserAuthzId, group, ServerAdminsAuthzId, update),
+    mv_oc_chef_authz:add_ace_for_entity(BifrostSuperuserAuthzId, actor, ErchefSuperuserAuthzId, group, ServerAdminsAuthzId, delete),
+    ok.
+
+%% Start vendor
+
+%% Vendored from mv_oc_chef_authz since not exported
```

Comment:
Any reason to not add the exports since we own it anyway? Ideally both in the mv_ version and the original so when we eventually switch away from vendored we don't break on it.